firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -129,16 +128,19 @@ def test_matching_text(self):
 
         autocompleteresults = self.browser.navbar.locationbar.autocomplete_results
         self.browser.navbar.locationbar.urlbar.send_keys(input_text)
-        self.wait_for_condition(lambda _: autocompleteresults.is_open)
+        self.wait_for_condition(lambda _: autocompleteresults.is_complete)
         visible_results = autocompleteresults.visible_results
         self.assertTrue(len(visible_results) > 0)
         for result in visible_results:
+            # check matching text only for results of type bookmark
+            if result.get_attribute('type') != 'bookmark':
+                continue
             title_matches = autocompleteresults.get_matching_text(result, "title")
             url_matches = autocompleteresults.get_matching_text(result, "url")
             all_matches = title_matches + url_matches
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
-                self.assertIn(match_fragment, (input_text, input_text.upper()))
+                self.assertIn(input_text, match_fragment.lower())
```

Comment:
Looks fine to me. Just a nit here, the value we want to compare it with is `input_text`. So it still has to be the second parameter. I will do this flip before I merge the commit.